.github/workflows/test_ipex.yml

```
@@ -18,6 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.8, 3.9]
+        transformers-version: [4.38.2, 4.41.2]
```

we also need to install this specific version of transformers so with something like `pip install transformers==${{ matrix.transformers }}` otherwise the latest version will always be installed

---

also could we use v4.38.0  directly
```suggestion
        transformers-version: [4.38.0, 4.41.2]
```